server/cmd/edgemesh-server/app/server.go

```diff
@@ -32,21 +32,21 @@ func NewEdgeMeshServerCommand() *cobra.Command {
 			flag.PrintFlags(cmd.Flags())
 
 			if errs := opts.Validate(); len(errs) > 0 {
-				klog.Fatal(util.SpliceErrors(errs))
+				klog.Exit(util.SpliceErrors(errs))
```

Comment:
I think we need the Fatal to log the detail error message

Reply:
I mean refer to the code of kubeedge. we have already do the validate. and klog.Exit will give the error info like bellowing:
F1126 04:35:54.842778       1 server.go:46] [
  kubeconfig: Invalid value: "/abc": kubeconfig not exist
]

So, we do not need the goroutine logs by using klog.Fatal.